README

```diff
@@ -66,6 +66,36 @@ In addition to the usual options, configure understands the following:
        this value is actually supplied as "\\\\".
 
 
+Building on macOS
+-----------------
+
+The following instructions apply to both Apple Silicon and Intel Macs.
```

Comment:
Please remove this line. It's great that the following instructions apply to all Macs, and soon presumably there will only be one type, so let's enjoy the fact that we don't have to mention it!